code/modules/error_handler/error_viewer.dm

```diff
@@ -0,0 +1,149 @@
+// Error viewing datums, responsible for storing error info, notifying admins
+// when errors occur, and showing them to admins on demand.
+
+// There are 3 different types used here:
+//
+// - ErrorCache keeps track of all error sources, as well as all individually
+//   logged errors. Only one instance of this datum should ever exist, and it's
+//   right here:
+
+#ifdef DEBUG
+var/global/datum/ErrorViewer/ErrorCache/error_cache = new()
```

Comment:
Comments do nothing, in most procs return does nothing.

How far are you going to go in your crusade?  PJB?  DO YOU HAVE NO SENSE OF DECENCY!?!